src/backend/columnar/columnar_customscan.c

```diff
@@ -824,6 +824,11 @@ ExtractPushdownClause(PlannerInfo *root, RelOptInfo *rel, Node *node)
 		}
 	}
 
+	if (IsA(node, ScalarArrayOpExpr))
+	{
+		return (Expr *) node;
```

Comment:
Should we still do any of the other checks on it? At least `!contain_volatile_functions()`?